app/scripts/modules/core/src/help/HelpField.tsx

```diff
@@ -1,8 +1,98 @@
+import * as React from 'react';
+import * as ReactGA from 'react-ga';
+import * as DOMPurify from 'dompurify';
+import autoBindMethods from 'class-autobind-decorator';
+import { $injector } from 'ngimport';
+
+import { HelpContentsRegistry } from 'core/help';
+import { HoverablePopover } from 'core/presentation';
+import { Placement } from 'core/presentation/Placement';
+
 export interface IHelpFieldProps {
   id?: string;
   fallback?: string;
   content?: string;
-  placement?: string;
+  placement?: Placement;
   expand?: boolean;
   label?: string;
 }
+
+export interface IState {
+  contents: React.ReactElement<any>;
+}
+
+@autoBindMethods
+export class HelpField extends React.Component<IHelpFieldProps, IState> {
```

Comment:
I reactipated this component, but left the angular `help-field` component intact